asmcli/lib/validate.sh

```diff
@@ -250,6 +251,26 @@ validate_ca_consistency() {
   fi
 }
 
+validate_no_ingress_gateway() {
+  local INGRESS_GATEWAY
+  INGRESS_GATEWAY="$(kubectl get deployments -n istio-system -l operator.istio.io/component=IngressGateways)"
+  if [[ -z "${INGRESS_GATEWAY}" ]]; then
+    return
+  fi
```

Comment:
Here's what I would check:

1. there's a service called "istio-ingressgateway" in the "istio-system" namespace
2. there's a deployment of the same name in the same namespace

Reply:
Done